build-release.sh

```diff
@@ -155,14 +145,24 @@ brew bottle \
 for file in *--*.bottle.tar.gz; do
   mv "$file" "$(echo "$file" | sed s/--/-/)"
 done
-aws s3 sync ./ s3://hhvm-downloads/homebrew-bottles/ --exclude '*' --include '*.bottle.tar.gz'
+if [ -z "$SKIP_PUBLISH" ]; then
+  aws s3 sync ./ s3://hhvm-downloads/homebrew-bottles/ --exclude '*' --include '*.bottle.tar.gz'
+fi
 
 PRE_BOTTLE_REV="$(git rev-parse HEAD)"
 
 function commit_and_push_bottle() {
   set -e
   git reset --hard "${PRE_BOTTLE_REV}"
   git pull origin master --rebase || (git rebase --abort; git reset --hard origin/master)
+
+  # delete any existing bottle references for the current OS version (from any
+  # potential previous bad/outdated builds)
+  source os-metadata.inc.sh
+  gsed -i "/sha256.\+ => :$OS_CODENAME/d" "${RECIPE}"
```

Comment:
you *might* need this before the `brew build` rather than just before brew bottle